Sources/BraintreeVenmo/BTVenmoRequest.swift

```diff
@@ -43,29 +43,43 @@ import Foundation
     public var displayName: String?
     
     /// Whether the customer's billing address should be collected and displayed on the Venmo paysheet.
-    /// Defaults to `false`
+    /// Defaults to `false`.
+    ///
+    /// If this value is set, `totalAmount` must also be set.
     public var collectCustomerBillingAddress: Bool = false
     
     /// Whether the customer's shipping address should be collected and displayed on the Venmo paysheet.
-    /// Defaults to `false`
+    /// Defaults to `false`.
+    ///
+    /// If this value is set, `totalAmount` must also be set.
     public var collectCustomerShippingAddress: Bool = false
```

Comment:
@khushboo18 - can you confirm that `totalAmount` is required when `collectCustomerBillingAddress` and `collectCustomerShippingAddress` are set?

I am seeing the same error for these params, but just want to confirm this is also expected.

Reply:
`totalAmount` is only _required_ when `transactionDetails` are set

Reply:
Merchants can set `collectCustomerBillingAddress` and `collectCustomerShippingAddress` without having to pass `totalAmount` as long as no other amounts or line items are being sent

Reply:
Got it - addressed! 7406aff3b7be8f51b1b50d618c8af4dd89c96efc